pyproject.toml

```
@@ -0,0 +1,16 @@
+[tool.poetry]
+name = "dj-registry"
```

The package's name should be the same as the current [`setup.py`](https://github.com/yychen/dj-registry/blob/master/setup.py).

```suggestion
name = "dj_registry"
```